src/intervals.h

```diff
@@ -223,11 +223,9 @@ set_interval_plist (INTERVAL i, Lisp_Object plist)
 /* If PROP is the `invisible' property of a character,
    this is 1 if the character should be treated as invisible,
    and 2 if it is invisible but with an ellipsis.  */
-
+extern int text_prop_means_invisible(Lisp_Object); // defined in intervals.rs
```

Comment:
Do `int` and `EmacsInt` always agree? I seem to recall this is not the case across platforms.

Reply:
Oops; changed to be EMACS_INT. In this case, the return value is always 0, 1, or 2, but in any case, the types probably should match.

Reply:
We have heard complaints in the past from people compiling on ARM that when the types don't match Rust will refuse to compile. Even though logically the types are the same.